excel_import_export/wizard/import_xlsx_wizard.py

```diff
@@ -0,0 +1,145 @@
+# Copyright 2019 Ecosoft Co., Ltd (http://ecosoft.co.th/)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html)
+
+from odoo import models, fields, api, _
+from odoo.exceptions import ValidationError, RedirectWarning
+
+
+class ImportXLSXWizard(models.TransientModel):
+    """ This wizard is used with the template (xlsx.template) to import
+    xlsx template back to active record """
+    _name = 'import.xlsx.wizard'
+
+    import_file = fields.Binary(
+        string='Import File (*.xlsx)',
+    )
+    template_id = fields.Many2one(
+        'xlsx.template',
+        string='Template',
+        required=True,
+        ondelete='set null',
+        domain=lambda self: self._context.get('template_domain', []),
+    )
+    res_id = fields.Integer(
+        string='Resource ID',
+        readonly=True,
+    )
+    res_model = fields.Char(
+        string='Resource Model',
+        readonly=True,
+        size=500,
+    )
+    datas = fields.Binary(
```

Comment:
s/datas/data because is plural form

Reply:
This one I just replicate the file content field of ir_attachment.datas (I think it is naming convention, isn't it?)

Reply:
non blocking on my side